### PR TITLE
feat: Add support for GRPC_ARG_ENABLE_HTTP_PROXY parameter

### PR DESCRIPTION
**Pull request by keskydi**
Add the support for the GRPC_ARG_ENABLE_HTTP_PROXY parameter in order to communicate between local processes

### CODE REVIEW

File: src/channel.rs [hunk 1]
```diff
@@ -295,6 +295,15 @@ impl ChannelBuilder {
         );
         self
     }
+    
+    /// If set to zero, disables use of http proxies.
+    pub fn enable_http_proxy(mut self, num: i32) -> ChannelBuilder {
```
Comment (BusyJay) [inline src/channel.rs:300]: How about using `bool`?

**Review by BusyJay — COMMENTED** [review]
Thanks! You need to sign off your commit.

File: src/channel.rs (hunk 1 above)
Comment (keskydi) [reply src/channel.rs:300]: The Options enum only define Integer, String and Pointeur

Comment (keskydi) [reply src/channel.rs]: modification done

**Review by BusyJay — APPROVED** [review]

**Comment by BusyJay** [comment]
You can use command `git commit --signoff` to signoff your commits. Note all commits need to be signed off.

**Comment by keskydi** [comment]
I had to fight with git to sign all my commits so i recreated a new pull request : https://github.com/tikv/grpc-rs/pull/574

**Comment by keskydi** [comment]
Add trouble with signing so i ended up doing another pull request : https://github.com/tikv/grpc-rs/pull/575

**Pull request closed without merging**